multply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
I think I found a typo.

```suggestion
    return x*y
```